packetbeat/protos/cassandra/internal/gocql/frame.go

```diff
@@ -443,7 +443,14 @@ func (f *Framer) parseResultPrepared() map[string]interface{} {
 
 	result := make(map[string]interface{})
 
-	result["prepared_id"] = string((f.decoder).ReadShortBytes())
+	uuid, err := UUIDFromBytes((f.decoder).ReadShortBytes())
```

Comment:
I am a little confused, the uuid need 16 bytes, but the prepare_Id is only two bytes here